src/components/Button/Button.js

```diff
@@ -2,7 +2,7 @@ import { observerListener } from '../../shared/observer';
 
 
 
-export const Button = ({ onClick, id, text, type, className }) => {
+export const Button = ( onClick, id, text, type, className ) => {
```

Comment:
Why did you changed the way that arguments are passed here?

Reply:
Ah I got some error so I tried fixed this somehow, I gonna change it to original version

Reply:
Done already